fsw/src/lc_events.h

```diff
@@ -401,7 +401,7 @@
  *  This event message is issued when a reset counters command has
  *  been received.
  */
-#define LC_RESET_DBG_EID 27
+#define LC_RESET_INF_EID 27
```

Comment:
Line 397 may need updating to `Type: INFORMATION`.

Reply:
I think I've updated both those changes now with the force-push Justin.
Thanks